pom.xml

```diff
@@ -1333,7 +1333,7 @@
                  <module>quarkus-vs-springboot</module>
                  <module>quarkus-jandex</module>
                  <module>spring-boot-modules/spring-boot-cassandre</module>
-                 <module>testing-modules/testing-assertions</module>
+		 <module>testing-modules/testing-assertions</module>
```

Comment:
This looks like only a whitespace change (from spaces to tabs). Since we should indent with spaces anyway and want to avoid whitespace-only changes, let's revert this one.